src/pages/workspace/WorkspaceOverviewPage.tsx

```diff
@@ -139,6 +142,8 @@ function WorkspaceOverviewPage({policyDraft, policy: policyProp, route}: Workspa
     const shouldShowAddress = !readOnly || !!formattedAddress;
     const {isAccountLocked, showLockedAccountModal} = useContext(LockedAccountContext);
     const [lastPaymentMethod] = useOnyx(ONYXKEYS.NVP_LAST_PAYMENT_METHOD, {canBeMissing: true});
+    const {isDismissed} = useDismissedUberBanners({policyID: policy?.id});
+    const shouldShowReceiptPartnersPromotionBanner = isBetaEnabled(CONST.BETAS.UBER_FOR_BUSINESS) && !policy?.areReceiptPartnersEnabled && !isDismissed && !readOnly;
```

Comment:
Could we remove this from here and instead control if the banner is shown or not exclusively from the banner component?
That would also allow us to remove the `useDismissedUberBanners` file (as the logic would only be used in one component, so we can just have the code in the component itself)